scripts/generate-config.js

```diff
@@ -126,7 +126,6 @@ commands:
                       FROM << parameters.imageName >>
                       RUN echo "current user: $(whoami)"
                       ENV CI=1
-                      WORKDIR /app
```

Comment:
These keep getting removed/added. Why is that?

Reply:
I did it because the build was failing and Zach suggested undoing the changes in his previous PR.

![image](https://user-images.githubusercontent.com/3003404/154148293-48f3685c-e9f7-41ba-a9ba-51fef454da47.png)

@flotwig - can you comment on this in more detail?

Reply:
Read the PR comment for 9.4.1 included image for some more context: https://github.com/cypress-io/cypress-docker-images/pull/587

So... for 9.4.1's image we managed to fix the Circle machine issue in #587, but we did it incorrectly, by manually editing `circle.yml`. Those changes didn't make it into `generate-image.js`, which is why @BlueWinds had to fix `generate-image.js` to get the fixes from #587 for this PR.

`generate-image.js` also had a no-op change for #587 where we modified that file, but did not use it to regenerate `circle.yml`, so that change was meaningless and probably wrong, which is why I said to revert it. 

Hope this clears it up - lmk if I can add add'l clarity.

Reply:
@flotwig @BlueWinds thanks so much for the context. That really helps. 

I keep seeing this `WORKDIR` added and removed over and over so I assume that could come back into play at some point, but the other changes look great.

Reply:
I am not sure why `WORKDIR` goes away and returns. I believe it had to be added in the first place to fix either `apt update` or `npm install`, can't remember which though.